src/content/en/tools/lighthouse/audits/required-aria-attributes.md

```diff
@@ -0,0 +1,43 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Elements With ARIA Roles Have Required Attributes" Lighthouse audit.
+
+{# wf_updated_on: 2017-01-17 #}
+{# wf_published_on: 2017-01-17 #}
+
+# Elements With ARIA Roles Have Required Attributes  {: .page-title }
+
+## Why the audit is important {: #why }
+
+The `role` attribute helps screen readers understand the purpose of an ARIA
+widget. Each `role` has a set of required, additional attributes that
```

Comment:
I think many roles do not have any required attributes so suggest rephrasing to "A `role` may also have a set of required, additional attributes".